WORKSPACE

```diff
@@ -79,3 +79,9 @@ docs_deps()
 load("@docs//:docs_transitive_deps.bzl", docs_transitive_deps = "transitive_deps")
 
 docs_transitive_deps(is_top_level = True)
+
+# For the hello_uses_cargo_manifest_dir example.
+local_repository(
```

Comment:
Why does this need to be a repository? And if it needs to be a repository, could it not be put into `example_deps` similar to how `rules_proto` was added to `docs/docs_transitive_deps.bzl`?
https://github.com/bazelbuild/rules_rust/blob/master/docs/docs_transitive_deps.bzl#L20-L31

Reply:
Great questions!

It needs to be a repository because that's specifically the case we are testing: something that depends on an external target that uses the `CARGO_MANIFEST_DIR` environment variable. If the rules is not external (ie. not in a separate repo) it won't reproduce the issue that this commit is fixing.

I can definitely put the deps in a separate bzl and will do so. Unfortunately I won't be able to reuse that `.bzl` file in `examples/WORKSPACE` because the `path` attribute has to be different, but it will still be a nice improvement to avoid polluting the root `WORKSPACE` 👍

Reply:
You could use a bool (`is_top_level`) similar to the snippet I sent you to toggle between the paths. Would that not be sufficient?

Reply:
You're totally right, I should have read your comment more carefully. Amended as suggested!